lnwallet/fee_estimator.go

```diff
@@ -458,3 +475,270 @@ func (b *BitcoindFeeEstimator) fetchEstimate(confTarget uint32) (SatPerKWeight,
 // A compile-time assertion to ensure that BitcoindFeeEstimator implements the
 // FeeEstimator interface.
 var _ FeeEstimator = (*BitcoindFeeEstimator)(nil)
+
+// WebAPIFeeEstimator is an implementation of the FeeEstimator interface that
+// queries an HTTP-based fee estimation from an existing web API.
+type WebAPIFeeEstimator struct {
```

Comment:
Awesome! 😀 

Would be cool if you could add this fee estimator in its own commit.

Reply:
fixed heh